### ISSUE TITLE
Dev marketing Thursday 1/31 Social Venue Ideas

### ISSUE THREAD

**Issue by haleygirl** [github/git-merge#183]
@DEGoodmanWilson 

Here are few cool ones from what I could tell. 
You'll need to contact them directly to see if they were open to private events, some appeared to be more catered for them and others not so much.

[Cantillon Brewery](https://www.cantillon.be)
* Museum and Brewery combination. Looks like it's used to having groups touring, so maybe a spot worthy of our size. 
* [Trip Advisor Information](https://www.tripadvisor.com/Attraction_Review-g188644-d245729-Reviews-Cantillon_Brewery-Brussels.html) (w/ better pictures/reviews)
* No available information online regarding private events
* Contact: museum@cantillon.be 

[Cafe La Brocante](https://cafelabrocante.skyrock.com/)
* [Trip Advisor Information](https://www.tripadvisor.com/Restaurant_Review-g188644-d1499987-Reviews-Cafe_La_Brocante-Brussels.html)
* Can't find contact (email) information but phone number in trip advisor link.
* Locals bar.
* The do live music (from looking at photos), so maybe big enough for a larger party? 

[Brasserie de l'Ermitage](https://www.facebook.com/ermitagenanobrasserie/?fref=ts)
* Contact: info@ermitagenanobrasserie.be. The website isn't really working so they take you to the FB, you can reach out to them that way too.
* Larger space - existing space might be a good set up for larger parties

[Le Porte Noire](https://www.laportenoire.be/en)
* [Trip Advisor Information](https://www.tripadvisor.com/Restaurant_Review-g188644-d4830906-Reviews-La_Porte_Noire-Brussels.html)
* Enchanting tavern once served as a kitchen for the nuns of the Alexien Convent.
* "hardly a week goes by without live music, whisky tasting or a night of serious partying" from site.
* [Generic contact form](https://www.laportenoire.be/contact). Or try phone from trip advisor link.

I can dig more into the central areas if these are a no-go, but decided to start with spaces that are by the train station.

**Comment by haleygirl** [comment]
@DEGoodmanWilson 

Just wanted to offer a couple tips my events team has found helpful when selecting a venue we have yet to see:

* Google street view (where you can see the actual area surrounding the venue), in case it's not very welcoming
* Map distance to the nearby local transportation and make sure it's an agreeable walking distance.

Alaine recalls two years ago that the neighborhood by the venue wasn't very 'cozy' :) so we just want to make sure folks feel safe!

**Comment by DEGoodmanWilson** [comment]
- [X] Cantillon — doesn't remain open after 5, does not rent out the venue.
- [ ] La Brocante
- [ ] Brasserie de l'Ermitage
- [ ] Le Porte Noire

**Comment by DEGoodmanWilson** [comment]
@haleygirl How many people are we expecting?

**Comment by DEGoodmanWilson** [comment]
@haleygirl it looks like we're struck out—none of the four venues are willing to rent out to us :( We'll need to keep digging.

**Comment by haleygirl** [comment]
@DEGoodmanWilson OK I'll send some more central spots for you. It's hard to tell how many. It's the day before the big day...I figure everyone is in town and I don't believe we have any other sponsor events to break up the headcount. The event is for 350...150? I might just look into bigger event spaces...

@AlaineN do we have any data to help determine how many to expect?

**Comment by AlaineN** [comment]
I don't think we should expect everyone to show up. There might be one other sponsored event that comes together, and also people might have their own things they want to do. I would find a space for 150 and take RSVPs  plus a waitlist.

**Comment by DEGoodmanWilson** [comment]
So, at 150, we need to consider something other than cafes. Most cafes / bars will have space for at most 50 TBH. Large bars are not really a thing in this part of Europe.

So, as you dig @haleygirl, bear in mind that we should probably be considering a dedicated venue at this scale.

**Comment by AlaineN** [comment]
@DEGoodmanWilson that's really good to know. I think we should walk this back to something more feasible. We want your team to feel like you can manage logisitics on this on your own, same as you might for any Drinkup. If a space you like can only hold 50, then we can take RSVPs and leave it at that. I can encourage another sponsor to host the same night to divide up attendees if that's helpful.

**Comment by DEGoodmanWilson** [comment]
As far as _my_ time is concerned, it really only depends on what we want out of the night. If we just want folks to have a good time and associate that goodwill with the GitHub name, then I've managed parties as large as 600 people in the past, no problem. The primary issue, as far as I can tell is that we need a venue to hold everyone. I really don't know Brussels, and my network is not coming through for me 😢 so I feel a bit like we're groping around in the dark.

**Comment by DEGoodmanWilson** [comment]
(I'm _not at all opposed_ to limiting it to ~50 so we can get a cozier venue, BTW!)

**Comment by haleygirl** [comment]
@DEGoodmanWilson - I reached out to https://www.spacehuntr.com/ and they're putting together options for me. He said he could get me some quotes and pricing as early as EOD your time today.

**Comment by haleygirl** [comment]
@DEGoodmanWilson 

From the company: 

> We received some feedback of the venues and see herewith our initial proposal of 3 venues. 
Please let us know your thoughts and if we can help clarify anything. 

Options are here: [Github Proposal V1.03.pdf](https://github.com/github/git-merge/files/2700038/Github.Proposal.V1.03.pdf)

Feel free to reach out to Dietrich Moens directly at dmoens@spacehuntr.com and let him know you're working with me.

**Comment by AlaineN** [comment]
@DEGoodmanWilson quick bump on this :)

**Comment by DEGoodmanWilson** [comment]
Talked with Dietrich on the phone about our needs, and the options. He will come back to me with final quotes later today.

**Comment by haleygirl** [comment]
@DEGoodmanWilson 

Hey! We need to move quickly on the GitHub sponsored event & RSVP list -- Can you please open an issue at MOPS: https://github.com/github/marketing-operations/issues/new/choose

Since we are TBD on the location, just pick one for the issue so the event page link can be created. I have access in the backend to change it once we confirm. MOPS is OOO at a mini-summit and we definitely want to try to get this event advertised by end of next week if possible. 

Can you build it out by Friday AM (PST?) I'll give Court a heads up that it's coming. If you can't do it, I can do it. Just let me know today if you can open that issue or not.

**Comment by DEGoodmanWilson** [comment]
We are no longer TBD on the location! We'll be at [Scotts Bar](https://www.google.com/maps/place/Scott's+Bar/@50.8488145,4.3561231,15z/data=!4m2!3m1!1s0x0:0xd4cc4ef657fdb034?ved=2ahUKEwiSv5rExOXfAhUDYlAKHZ5CAeQQ_BIwCnoECAYQCA)

**Comment by DEGoodmanWilson** [comment]
https://github.com/github/marketing-operations/issues/1290

I am curious to know if we can find a way to limit guests to attendees of Git Merge…any ideas?
